tests/test_timeseries.py

```diff
@@ -152,8 +152,102 @@ def test_build_from_lists(self):
         self.assertEqual(self.cts1.timezone, pytz.timezone("UTC"))
         self.assertEqual(self.cts1.name, "Test a categorical time series")
         self.assertEqual(self.cts1.type, 'CatTimeSeries')
-        
-        
+
+    def test_add_constant(self):
+        # Given
+        test_df = pd.DataFrame(columns=['ts'], index=['2020-01', '2020-02', '2020-03'])
+        test_df.loc['2020-01'] = 1.
+        test_df.loc['2020-02'] = 2.
+        test_df.loc['2020-03'] = 3.
+        ts1 = ts.TimeSeries(data=test_df, tz="Europe/London", unit='£', name="Test a time series")
+
+        expected = [4., 5., 6.]
+
+        # When
+        sum_result = ts1 + 3
+        actual = list(sum_result.data)
+
+        # Then
+        self.assertListEqual(expected, actual)
+
+    def test_add_list(self):
+        # Given
+        test_df = pd.DataFrame(columns=['ts'], index=['2020-01', '2020-02', '2020-03'])
+        test_df.loc['2020-01'] = 1.
+        test_df.loc['2020-02'] = 2.
+        test_df.loc['2020-03'] = 3.
+        ts1 = ts.TimeSeries(data=test_df, tz="Europe/London", unit='£', name="Test a time series")
+
+        expected = [4., 6., 8.]
+
+        # When
+        sum_result = ts1 + [3, 4, 5]
+        actual = list(sum_result.data)
+
+        # Then
+        self.assertListEqual(expected, actual)
+
+    def test_add_ts(self):
+        # Given
+        test_df = pd.DataFrame(columns=['ts'], index=['2020-01', '2020-02', '2020-03'])
+        test_df.loc['2020-01'] = 1.
+        test_df.loc['2020-02'] = 2.
+        test_df.loc['2020-03'] = 3.
+        ts1 = ts.TimeSeries(data=test_df, tz="Europe/London", unit='£', name="Test a time series")
+
+        # create a ts with all ones
+        ts2 = ts1 * 0 + 1
```

Comment:
This is an original way to generate a series of 1s with the index of another series. I think it's great! But it makes me think also that it would be practical to have a function that generates a constant time series from the start. What do you think?